docs/ci-cd-environment/macos-xcode-11-image.md

```diff
@@ -1,17 +1,17 @@
-# macOS Mojave Xcode 10 Image
+# macOS Xcode 11 Image
```

Comment:
Rename to "macOS Catalina Xcode 11 Image" and than we can rename it back when we deprecate Mojave.